libs/output-mapping/src/Configuration/Table/BaseConfiguration.php

```diff
@@ -47,18 +50,114 @@ public static function configureNode(ArrayNodeDefinition $node): void
                 ->scalarNode('delete_where_column')->end()
                 ->arrayNode('delete_where_values')->prototype('scalar')->end()->end()
                 ->scalarNode('delete_where_operator')
-                    ->defaultValue('eq')
+                    ->defaultValue(self::DEFAULT_DELETE_WHERE_OPERATOR)
                     ->beforeNormalization()
                         ->ifInArray(['', null])
                         ->then(function () {
-                            return 'eq';
+                            return self::DEFAULT_DELETE_WHERE_OPERATOR;
                         })
                     ->end()
                     ->validate()
-                    ->ifNotInArray(['eq', 'ne'])
+                    ->ifNotInArray(self::ALLOWED_DELETE_WHERE_OPERATORS)
                         ->thenInvalid('Invalid operator in delete_where_operator %s.')
                     ->end()
                 ->end()
+                ->arrayNode('delete_where')
+                    ->prototype('array')
+                        ->children()
+                            ->scalarNode('changed_since')->end()
+                            ->scalarNode('changed_until')->end()
```

Comment:
neříkali jsme že changed_since/until sem zatím dávat nebudeme?

Reply:
Uvidim jak bude slozita implementace. Minimalne v config definition bych podporoval prave vsechno. At se to pak zas nemusi upravovat a na deseti mistech aktualizovat

Reply:
nj já když jsem to říkal že bych to tam rovnou přidal tak jste mi s odinem říkali že ne, že to je jen nastavení kdyby to někdo v budoucnu chtěl

jako za mě ta implementace je s minimálním rozdílem, takže já jsem pro to tu nechat a rovnou i implementovat no....

Reply:
co na to @odinuv ?

Reply:
kdo mlci souhlasi 🤷

Reply:
Jinak samotna implementace change since/until v te logice OM jsou dva ify a dve property. Ja bych to tam nechal i kdyz to nebude zatim v UI.

Reply:
jo souhlas 👍

Reply:
souhlasí